draft-ietf-scitt-architecture.md

```diff
@@ -486,7 +486,6 @@ Requesting a Receipt can result in the production of a new Receipt for the same
 A Receipt's verification key, signing algorithm, validity period, header parameters or other claims MAY change each time a Receipt is produced.
 
 Anyone with access to the Transparency Service can independently verify its consistency and review the complete list of Transparent Statements registered by each Issuer.
-However, the Registrations on a separate Transparency Service is generally disjoint, though it is possible to take a Transparent Statement (i.e. a Signed Statement with a Receipt in its unprotected header, from the first Transparency Service) and register it on another Transparency Service, where the second Receipt will be over the first Receipt in the unprotected header.
```

Comment:
+1 to removing this. Registering signed statements across multiple transparency services are independent actions. While it's possible to attach multiple receipts, the VDS does nothing with the receipts in the unprotected headers. 
A TS can store the signed statement, with the receipts, but they're "unprotected".